examples/autoscaler-webhook/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	v1alpha1 "agones.dev/agones/pkg/apis/stable/v1alpha1"
+	"agones.dev/agones/pkg/util/runtime" // for the logger
+)
+
+// Constants which define thresholds to trigger scalling up and scale factor
+const (
+	replicaUpperThreshold = 0.7
+	replicaLowerThreshold = 0.3
+	scaleFactor           = 2
+	minReplicasCount      = 2
+)
+
+// Variables for the logger
+var (
+	logger = runtime.NewLoggerWithSource("main")
+)
+
+// Main will set up an http server and three endpoints
+func main() {
+	// Serve 200 status on / for k8s health checks
+	http.HandleFunc("/", handleRoot)
+
+	// Serve 200 status on /health for k8s health checks
+	http.HandleFunc("/health", handleHealth)
+
+	// Return the target replica count which is used by Webhook fleet autoscaling policy
+	http.HandleFunc("/scale", handleAutoscale)
+
+	if err := http.ListenAndServe(":8000", nil); err != nil {
+		logger.WithError(err).Fatal("HTTP server failed to run")
+	} else {
+		logger.Info("HTTP server is running on port 8000")
```

Comment:
you don't actually reach this line when the server is running, so I would remove it and log before with `Starting HTTP server ....`